xcm/pallet-xcm-benchmarks/src/generic/benchmarking.rs

```diff
@@ -497,11 +497,13 @@ benchmarks! {
 	}
 
 	export_message {
+		let (network, destination) = T::export_message_destination().map_err(|_| BenchmarkError::Skip)?;
```

Comment:
use the error specified by the trait fn implementation:

```suggestion
		let (network, destination) = T::export_message_destination()?;
```

Reply:
benchmark export_message was reverted